src/molecules/formfields/SliderField/Readme.md

```diff
@@ -18,13 +18,15 @@
       }
 
       render() {
+        const { value } = this.state;
+
         return (
           <SliderField
             sliderValues={
               [ 1000, 5000, 25000, 500000 ]
             }
             input={{
-              value: this.state.value,
+              value,
```

Comment:
I'm not sure this is right, since the previous code was a prop assignment and this is just a value now, you could do `const { currentValue } = this.state;` and then `value: currentValue`

Reply:
there are a few other times this refactor happened as well, I just noticed it